components/blitz/src/omero/cmd/graphs/DuplicateI.java

```diff
@@ -530,7 +802,7 @@ public void adjustGraphPolicy(Function<GraphPolicy, GraphPolicy> adjuster) {
 
     @Override
     public int getStepProvidingCompleteResponse() {
-        return dryRun ? 0 : 5;
+        return dryRun ? 0 : 6;
```

Comment:
How does the `6` here match with https://github.com/openmicroscopy/openmicroscopy/pull/4546/files#diff-5e2f58efa615b81632d4ab5b9186dd45R187 ?